tx-parser/src/lib.rs

```diff
@@ -13,12 +13,17 @@
 extern crate edn;
 extern crate combine;
 extern crate mentat_tx;
+extern crate mentat_parser_utils;
+
+#[macro_use]
+extern crate mentat_query_parser;
```

Comment:
Further to the above, I expect the macros to come from the parsing utilities crate.